frontend/tsconfig.json

```diff
@@ -17,7 +17,7 @@
     "skipLibCheck": true,
     "sourceMap": true,
     "baseUrl": ".",
-    "typeRoots": ["./node_modules/@types"],
+    "typeRoots": ["./node_modules/@types", "./src/components/nodes/Datamaps"],
```

Comment:
Let's not do this. Instead of adding the `index.d.ts` and listing it here, let's just add that line to `frontend/next-env.d.ts`.